sklego/preprocessing/outlier_remover.py

```diff
@@ -42,6 +42,24 @@ def fit(self, X, y=None):
         -------
         self : OutlierRemover
             The fitted transformer.
+       
+        Example
+        -------
+        ```py
+        from sklego.preprocessing import OutlierRemover
+        from sklearn.ensemble import IsolationForest
+
+        np.random.seed(0)
+        X = np.random.randn(10000, 2)
+
+        isolation_forest = IsolationForest()
+        isolation_forest.fit(X)
+        detector_preds = isolator_forest.predict(X)
+
+        outlier_remover = OutlierRemover(isolation_forest, refit=True)
+        outlier_remover.fit(X)
+        X_trans = outlier_remover.transform_train(X)
+        ```
```

Comment:
```suggestion
        Example
        -------
        ```py
        from sklego.preprocessing import OutlierRemover
        from sklearn.ensemble import IsolationForest

        np.random.seed(0)
        X = np.random.randn(10000, 2)

        isolation_forest = IsolationForest()
        isolation_forest.fit(X)
        detector_preds = isolator_forest.predict(X)

        outlier_remover = OutlierRemover(isolation_forest, refit=True)
        outlier_remover.fit(X)
        ```
```